.eslintrc.json

```
@@ -0,0 +1,15 @@
+{
+	"env": {
+		"es6": true,
+		"node": true
```

```suggestion
		"browser": true,
		"node": true,
		"es2020": true
```

You have a mix between es6 (emacVersion: 2015) and es10 (emacVersion 2019). Define both at 2020.